test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -347,19 +423,14 @@ private void GenerateManifestAndValidateSuccess(string testFolderPath, string ma
         Assert.IsTrue(File.Exists(jsonFilePath));
         Assert.IsTrue(File.Exists(shaFilePath));
 
-        // Check that manifestFolderPath is the only folder in the directory
-        var directories = Directory.GetDirectories(Path.Combine(testFolderPath, ManifestRootFolderName));
-        Assert.AreEqual(1, directories.Length, "There should be only one folder in the test directory.");
```

Comment:
Why remove this check? I assume now we expect for there to be 2 dirs (one for 2.2 and one for 3.0), is that right? If so, let's assert that.

Reply:
Turns out the reason `_manifest` had `spdx_2.2` and `spdx_3.0` was because of the DataRow on this unit test which meant the folder structure was not cleaned up properly. I've separated the tests and added back the assertion that there should only be one folder within `_manifest`.